lib/application.ts

```diff
@@ -703,8 +710,8 @@ export class SNApplication {
   /**
    * Returns true if there is an encryption source available
    */
-  public isEncryptionAvailable() {
-    return this.hasAccount() || this.hasPasscode();
+  public async isEncryptionAvailable() {
+    return !isNullOrUndefined(this.getUser()) || this.hasPasscode();
```

Comment:
Why is this undoing changes @radko93 recently introduced?

Reply:
Seems like a mistake

Reply:
Sorry about that, fixed :+1: